conn_test.go

```diff
@@ -349,15 +350,21 @@ func TestCancel(t *testing.T) {
 type testQueryObserver struct {
 	metrics map[string]*hostMetrics
 	verbose bool
-	logger  StdLogger
+	logger  StructuredLogger
 }
 
 func (o *testQueryObserver) ObserveQuery(ctx context.Context, q ObservedQuery) {
 	host := q.Host.ConnectAddress().String()
 	o.metrics[host] = q.Metrics
 	if o.verbose {
```

Comment:
Since we have log levels now, I think we might be able to do away with this verbose mode. But that could be in a follow-up PR.

Reply:
Done